libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -180,7 +180,7 @@ static AwsIotProvisioningError_t _serializeCreateKeysAndCertificateRequestPayloa
     /* Encode an empty map container (Diagnostic notation as "{}"") .*/
     serializerStatus = _pAwsIotProvisioningEncoder->openContainer( pOutermostEncoder,
                                                                    &emptyPayloadEncoder,
-                                                                   _numCertFromCsrMapEntries );
+                                                                   _numKeysAndCertMapEntries );
 
```

Comment:
How was it compiling before?

Reply:
`_numCertFromCsrMapEntries` is a valid constant, but this specific function is related to the "KeysAndCertificate" operation.
I caught the error on my machine as I used the `-Wall` flag for building (This specifically was raised by the `-Wunused-const-variable=` flag